.github/ISSUE_TEMPLATE/patternfly-issue-template.md

```diff
@@ -0,0 +1,17 @@
+---
+name: PatternFly issue template
+about: Please populate the PatternFly issue template so we can learn more about your
+  needs.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+### Describe the issue. What are the expected and unexpected behavior categories?
```

Comment:
I don't like this wording. As a dev, I'm not sure what a behaviour "category" is.